src/components/tooltip/index.js

```diff
@@ -47,23 +42,30 @@ const propTypes = {
       PropTypes.number,
     ]).isRequired,
   })),
+  typography: PropTypes.object.isRequired,
 }
 
 const Tooltip = ({
   label,
   color,
   display,
+  typography,
 }) => (
   <TooltipWrapper>
     <TooltipHeader>
       <TooltipNode background-color={ color } />
-      <TooltipLabel>
+      <TooltipLabel 
+        style={ typography }
+      >
```

Comment:
if we aren't utilizing the `styled` interface, then might as well use the raw `<strong>` tag

on the other hand, why not parameterize the `styled` component to accept the custom styling, instead of inline styles? it's a performance inhibitor as the style object gets recomputed on every render, while the use of a css-in-js solution (such as goober `styled`) would compute it once and serve it as static thereafter.

Reply:
ok, will do.. Shane preferred using the inline style, if I got his comments right, for chart-wrapper component. Then I guess I should change it back there as well....

Reply:
https://github.com/EQWorks/chart-system/pull/115#discussion_r483628661 my first suggestion was to spread the props inside the styled component. For the chart-wrapper, since it's the top-level component, I figured it would change relatively little, but I guess it's just a best practice to never? Sorry for the confusion.

Reply:
i see, then these are not contradicting anyways